src/time_integration/time_integration.jl

```diff
@@ -27,6 +27,17 @@ function DiffEqBase.get_tstops_max(integrator::AbstractTimeIntegrator)
     return maximum(get_tstops_array(integrator))
 end
 
+function finalize_callbacks(integrator::AbstractTimeIntegrator)
+    @unpack alg = integrator
```

Comment:
You don't need this line, right?

Reply:
Yes, you are right. Should be fixed now. Thanks!